src/main/resources/META-INF/plugin.xml

```diff
@@ -199,7 +199,7 @@
             from `Search Everywhere` and by link displayed in graph table when `machete` file
             is empty or absent -->
         <action id="GitMachete.DiscoverAction"
-                class="com.virtuslab.gitmachete.frontend.actions.graphtable.DiscoverAction"
+                class="com.virtuslab.gitmachete.frontend.actions.unanchored.DiscoverAction"
```

Comment:
One file to update is also docs/plugins.jetbrains.com/general_usage_instructions.html - there's still a TODO for #176 there :/